library/spdm_requester_lib/libspdm_req_get_capabilities.c

```diff
@@ -137,6 +153,22 @@ static bool validate_responder_capability(uint32_t capabilities_flag, uint8_t ve
         }
     }
 
+    /* Checks specific to 1.3. */
```

Comment:
Public key and multi-key are mutually exclusive. If one is set then the other must be cleared.

Reply:
Really? Would you please point to me the text in spec?

Reply:
I believe that was the intent of the specification. If `PUB_KEY_ID_CAP == 1 && MULTI_KEY_CAP != 0` is legal then there needs to be a lot of clarifications and restrictions.

Reply:
Filed issue as https://github.com/DMTF/SPDM-WG/issues/3277.

Reply:
I think we can add the check, **after** the spec clarification.

I prefer to keep this patch as is here.